Pcap++/header/PcapLiveDevice.h

```diff
@@ -578,4 +579,10 @@ namespace pcpp
 		pcap_t* doOpen(const DeviceConfiguration& config);
 	};
 
+	//route message struct for communication in BSD / APPLE device
+	struct BSDRoutingMessage{
+		struct	rt_msghdr header;
+		char	messageSpace[512];
+	};
```

Comment:
The struct definition in the `route.c` source code of macOS follows C language conventions
`struct {
	struct	rt_msghdr m_rtm; 
	char	m_space[512];
} m_rtmsg;
`
I choose current naming to try to suit Pcap variables' name style, do you have any suggestions on the naming? @tigercosmos